tests/roles/manila_adoption/tasks/main.yaml

```diff
@@ -36,3 +36,13 @@
   register: manila_responding_result
   until: manila_responding_result is success
   retries: 15
+
+- name: Create default share type
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    alias openstack="oc exec -t openstackclient -- openstack"
+    ${BASH_ALIASES[openstack]} share type create {{ share_type_name }} {{ driver_handles_share_servers }}
```

Comment:
should we provide some default for that thought like there https://github.com/marios/data-plane-adoption/blob/aa7c61287ae041d7a6f79f14ea4ca471ef1e324a/tests/roles/manila_adoption/defaults/main.yaml#L1 for share_type_name

or do we need to make that configurable https://github.com/marios/data-plane-adoption/blob/aa7c61287ae041d7a6f79f14ea4ca471ef1e324a/tests/roles/manila_adoption/templates/manila_cephfs.yaml.j2#L41

Reply:
I think having the command: `openstack share type create default false` is sufficient and generic enough for now.
We can think about customizing it in the context of unijobs implementation or multibackend, but right now I simply hardcoded `default` and `false` because these two parameters, in the adoption context, are always valid.
I'm planning to expand this part when the CI coverage will be extended further.